### PR TITLE
movie details

### PR DESCRIPTION
**Pull request by gregoryjflaherty**
## Description of Changes
* Edit viewing party dashboard
* Add tests for models
* Add instance methods for models 

## Concerns (tags recommended)
Merge conflicts

### CODE REVIEW

File: app/facades/movie_facade.rb [hunk 1]
```diff
@@ -7,6 +7,7 @@ def self.get_10_cast(api_id)
     MovieService.get_cast(api_id)[0..9]
```
Comment (conorbarthel) [inline app/facades/movie_facade.rb:7]: This works, I wonder if this is preferable to modifying the results in the model.

File: spec/models/party_spec.rb [hunk 2]
```diff
@@ -7,7 +7,29 @@
...
   end
 
-  describe "validations" do
+  describe ".instance methods" do
+    before(:each) do
+      @jax = User.create!(name: "Jackson", email: "j@jmail.com")
+      @lady = User.create!(name: "LadyBug", email: "l@lmail.com")
 
+      @movie_1 = Movie.create!(api_id: 550)
+      @movie_2 = Movie.create!(api_id: 600)
```
Comment (conorbarthel) [inline spec/models/party_spec.rb:16]: nice use of movies in a limited capacity.

**Review by conorbarthel — APPROVED** [review]
Looks super solid, may need to add validations as we go forward.

File: app/facades/movie_facade.rb (hunk 1 above)
Comment (gregoryjflaherty) [reply app/facades/movie_facade.rb:7]: Probably not actually - we should see if this would go better elsewhere , I would imagine model level

**Pull request merged**